scripts/check-data-freshness.ts

```diff
@@ -0,0 +1,97 @@
+import axios from 'axios';
+import axiosRetry from 'axios-retry';
+import { test } from 'tap';
+import { PROGRAMS } from '../src/data/programs';
+
+// Returns the status obtained by trying to request the URL content.
+async function checkUrlDataAvailability(
+  link: string,
+): Promise<number | undefined> {
+  // Set the number of retries to 3 for network errors or those in the 5xx range.
+  axiosRetry(axios, {
+    retries: 3,
+  });
+  const content = await axios({
+    method: 'get',
+    url: link,
+    timeout: 10000,
+  }).catch(function (error) {
+    // If a non-2xx response status exists, return it.
+    if (error.response) {
+      /**
+      console.log(
+        source,
+        ' has an non-OK status code of: ',
+        error.response.status,
+        'with URL',
+        link,
+      );
```

Comment:
Could you remove this commented-out code? I wasn't sure the best place to add the responses, but I think tap's comments are good enough. (here and below)